Ant.cpp

```diff
@@ -1,203 +1,97 @@
 /**************************************************************************************************
-** Program name:Group Project - Predator-Prey Game
+** Program Name:Group Project - Predator-Prey Game
 ** Author:	Group 4
+**		      Herbert Diaz  <diazh@oregonstate.edu>
```

Comment:
I think we should remove names and info and leave it more general since we all generally put work into the different classes/functions

Reply:
I pulled out the Author information aside from Group 4.